src/daft-core/src/series/ops/mod.rs

```diff
@@ -88,3 +89,36 @@ macro_rules! py_binary_op_utilfn {
 }
 #[cfg(feature = "python")]
 pub(super) use py_binary_op_utilfn;
+
+#[cfg(feature = "python")]
+macro_rules! py_membership_op_utilfn {
```

Comment:
why does this have to be a macro? Can this be a function?

Reply:
changed to function